.travis.yml

```diff
@@ -1,19 +1,21 @@
+sudo: false
 language: python
-
+cache: pip
 python:
-  - "3.4"
-
-install:
-  - "pip install flake8 jsonschema"
-
-script:
-  - "flake8"
-  - "python compile.py"
-  - "mkdir build; mv extensions.* build"
-
+  - "3.6"
+addons:
+  apt:
+    packages:
+    - lftp
 deploy:
```

Comment:
Yeah, but for comparing cross-repository alignment of .travis.yml files, `deploy` is identical but `install` and `script` are not, so I think it's a fair compromise.